modules/httpcache/httpcache_test.go

```diff
@@ -24,6 +25,17 @@ func (m mockFileInfo) ModTime() time.Time { return time.Time{} }
 func (m mockFileInfo) IsDir() bool        { return false }
 func (m mockFileInfo) Sys() interface{}   { return nil }
 
+func countFormalHeaders(h http.Header) (c int) {
```

Comment:
Yeah, I can see that, but maybe we should rename the `Formal` to `Official` or `Standard`.

Reply:
I didn't use "standard" because we may want some customization in future, which contains our customized headers should be counted. It's just a personal preference, and this function just appears in the unit test privately, so I think this name doesn't matter much.